R/lookup.R

```diff
@@ -40,6 +40,7 @@ equity_market_filter_lookup <- "GlobalMarket"
 p4i_p4b_sector_technology_lookup <- tibble::tribble(
   ~sector_p4b,   ~technology_p4b,             ~sector_p4i,    ~technology_p4i,
   "automotive",  "electric",                 "Automotive",   "Electric",
+  "automotive",  "fuelcell",                 "Automotive",   "FuelCell",
```

Comment:
1. change of this mapper makes calculation of ETP2017 fail
2. This introduces an inconsistency between this mapper and its mirror in st data prep repo. this is a bit of a problem. In data prep we use this mapper to infer if sectors are complete

I synch we should quickly synch on how to resolve this.